test/remote/gateways/remote_rapyd_test.rb

```diff
@@ -478,4 +478,26 @@ def test_successful_subsequent_purchase_stored_credential_payment_redirect_url
     assert_success response
     assert_equal 'SUCCESS', response.message
   end
+
+  def test_successful_purchase_with_fx_fields_with_currency_exchange
+    @options[:pm_type] = 'gb_visa_card'
+    @options[:currency] = 'GBP'
+    @options[:requested_currency] = 'USD'
+    @options[:expiration_card] = Time.mktime(2024, 1, 2, 12, 0, 0)
```

Comment:
❓ I have a doubt,  what is the use for expiration_card? regarding we are in 2025, should the year be 2025?

Reply:
Hi @gasb150 the `expiration_card` field is required to make a purchase with fx when the currency is different to 'EUR', I will update the Time, thanks